### DISCUSSION TITLE
unable to ignore module hosted in Terraform Cloud

### DISCUSSION THREAD

**Discussion by jaywithersbrewin** [terraform-linters/tflint#1916] [Q&A] [answered]
Hi!

I'm trying to ignore a module that's hosted in Terraform Cloud, to avoid:
**Warning: module "naming_standard" should specify a version (terraform_module_version)**

The module source is "app.terraform.io/rbcbrewindolphin/naming_standard/bd"

I've tried various combinations of the module name (in config/ignore_module block) but still getting the warning.#

TIA

**Comment by bendrucker** [comment, marked as answer]
That feature is for ignoring errors that arise from _within_ specified modules. This rule doesn't evaluate the module, just the existence of `module` blocks' `version` attributes.

You'll need to use a `tflint-ignore` annotation on the `module` block itself.

**Comment by jaywithersbrewin** [comment]
Thanks for clearing it up, Ben.
